django/contrib/gis/db/models/__init__.py

```diff
@@ -11,4 +11,4 @@
 from django.contrib.gis.db.models.fields import (  # NOQA
     GeometryField, PointField, LineStringField, PolygonField,
     MultiPointField, MultiLineStringField, MultiPolygonField,
-    GeometryCollectionField)
+    GeometryCollectionField, RasterField)
```

Comment:
How about checking for raster support on initialization of the field?

https://github.com/django/django/pull/4339/files#diff-466a45d312b8b1061be8209a47db2e40R169